tests/backends/cursors/postgres.py

```diff
@@ -0,0 +1,32 @@
+try:
+    import psycopg2
+    import psycopg2.extensions
+    HAS_PSYCOPG2 = True
+except ImportError:
+    HAS_PSYCOPG2 = False
+    pass
+
+if HAS_PSYCOPG2:
```

Comment:
Please change to `@unittest.skipUnless(connection.vendor == 'postgresql', ...`  instead of `try/except`.